gcp_variant_transforms/transforms/vcf_field_conflict_resolver.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
```

Comment:
I think it makes more sense to move this out of the transforms directory to "libs". There is nothing Beam specific in this module (which is a good thing), is it?

Reply:
Done!